cloud-config/README.md

```diff
@@ -0,0 +1,82 @@
+# Organization deployment configuration and build files
+## Directories
+    * config-build-files
+    * deploy-run-scripts
+    * prebuilt-config-yamls
+
+
+# Primary Deployments
+* Development Deployment or Demo: Single instance with front, postgres, es data, es head 
+* Elect Cluster: Single instance with frontend and postgres.  5 Node cluster with self elected head node.
+* Wait Cluster: Single instance with frontend and postgres.  5 Node cluster with 6th instance for head node.
+
+## Dev Deployment
+    * Made       Prebuild: $bin/deploy -n dev-conf --save-config-name 20191002
+        # Wrote new config to ./cloud-config/prebuilt-config-yamls/20191002-demo.yml
+    * Diff Build/Prebuild: $bin/deploy -n dev-conf --use-prebuilt-config 20191002-demo --diff-configs
+    * Deploy     Prebuild: $bin/deploy -n dev-conf --use-prebuilt-config 20191002-demo
+    * Deploy        Build: $bin/deploy -n dev-conf
```

Comment:
should -n dev-conf be the default?

Reply:
I'm not trying to explain the deploy command in this readme.  I think people understand it well enough anyway.  Or get errors for trying to create an instance with the same name!

Reply:
I fixed the ticket number name also.